src/borg/algorithms/lz4/lib/lz4.c

```diff
@@ -986,7 +986,7 @@ LZ4_FORCE_INLINE int LZ4_compress_generic(
 _next_match:
         /* at this stage, the following variables must be correctly set :
          * - ip : at start of LZ operation
-         * - match : at start of previous pattern occurence; can be within current prefix, or within extDict
+         * - match : at start of previous pattern occurrence; can be within current prefix, or within extDict
```

Comment:
all algorithms/* stuff needs to be fixed upstream. i just copy the files from there now and then.

https://github.com/lz4/lz4
https://github.com/facebook/zstd
https://github.com/Cyan4973/xxHash

Reply:
> all algorithms/* stuff needs to be fixed upstream. i just copy the files from there now and then.

You are right. I can drop this part of the patch.
I'm going to submit upstream (but not all project are happy to receive/merge typo fixes).
Anyway, let's see.

Reply:
they do cool stuff, so i'ld hope they also appreciate it.

Reply:
fixed here. still needs to get fixed upstream.